phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java

```diff
@@ -127,6 +127,7 @@ private void populateRepositoryGrid() {
         this.addButton.setText(tr("Add"));
         this.addButton.setOnAction((ActionEvent event) -> {
             AddRepositoryDialog dialog = new AddRepositoryDialog();
+            dialog.initOwner(this.getParent().getScene().getWindow());
```

Comment:
Didn't check if the stylesheet is applied but if you do not have this (probably yes), the dialog does not use the correct window icon.